src/Files.App/ServicesImplementation/Settings/AppearanceSettingsService.cs

```diff
@@ -117,6 +169,10 @@ public void ReportToAppCenter()
             Analytics.TrackEvent($"{nameof(ShowFileTagsSection)}, {ShowFileTagsSection}");
             Analytics.TrackEvent($"{nameof(PinRecycleBinToSidebar)}, {PinRecycleBinToSidebar}");
             Analytics.TrackEvent($"{nameof(UseCompactStyles)}, {UseCompactStyles}");
-        }
-    }
+			Analytics.TrackEvent($"{nameof(ShowFoldersWidget)}, {ShowFoldersWidget}");
+			Analytics.TrackEvent($"{nameof(ShowRecentFilesWidget)}, {ShowRecentFilesWidget}");
+			Analytics.TrackEvent($"{nameof(ShowDrivesWidget)}, {ShowDrivesWidget}");
+			Analytics.TrackEvent($"{nameof(ShowBundlesWidget)}, {ShowBundlesWidget}");
```

Comment:
The `ReportToAppCenter` method is no longer used and was accidentally left in, can you remove it?